README.md

```diff
@@ -23,7 +23,7 @@ If you want to serve app with your configuration rename config-template.yml to c
 Values descriptions you can find inside config-template.yml.
 
 Afterwards run it with:
-> poetry run flask --app 'goodmap.goodmap(config_path="<path-to-your-config>")' --debug run
+> poetry run flask --app 'goodmap.goodmap:create_app(config_path="PATH-TO-YOUR-CONFIG")' --debug run
```

Comment:
Good finding, but it comes with stupid question...

Stupid question, but I can't find good answer for 'how should argument for python command look like in markdown".

Did you find it maybe?